### ISSUE TITLE
Kernel panic during DHCP process

### ISSUE THREAD

**Issue by realJustinLee** [coolsnowwolf/lede#9464]
## Kernel panic during DHCP process

Q：是否搜索了issue (使用 "x" 选择)
* [x] 没有类似的issue

## 2. 详细叙述
The kernel panic yields during the DHCP process in the latest build.
A detailed system log could be found in the screenshot below.

### (1) 具体问题
While the router was doing DHCP, especially renewing the lease, you would see the router yields a kernel panic and reboots itself.

### (2) 路由器型号和固件版本
general x86
latest source code build

### (3) 详细日志
<img width="1024" alt="Screen Shot 2022-06-01 at 21 49 39" src="https://user-images.githubusercontent.com/17794470/171421602-d56e8ed9-962b-4a3a-9266-48cf16922f71.png">

**Comment by 097** [comment]
I encountered a similar issue. The system reboot unexpectedly within about one miniute after services in the user space are started. The Openwrt system image is built by myself, with the latest source code. The last commit info of source code is as below:

commit 2b57a426cba1f83f84e04c4a87205bb592735faa (HEAD -> master, origin/master, origin/HEAD)
Author: lean <coolsnowwolf@gmail.com>
Date:   Tue May 31 23:00:41 2022 +0800

    kernel: backport flow offload fixes

I installed the Openwrt in PVE.
